frontend/src/modules/resourceprogress/components/ResourceProgress.js

```diff
@@ -62,32 +66,38 @@ export class ResourceProgressBase extends React.Component<Props, State> {
             <aside className="menu">
                 <div className="main">
                     <header>
-                        <h2>All strings <span className="value">{ total }</span></h2>
-                        <h2 className="small">Unreviewed <span className="value">{ unreviewed }</span></h2>
+                        <h2>
+                            All strings
+                            <span className="value">{ Number(total).toLocaleString('en-US') }</span>
```

Comment:
That seems fair enough to me for now, but once we have support for more locales in Pontoon, I think I'd like to make this internationalized. Maybe we should file a P5 bug about this?

Reply:
Also, this pattern is repeated a lot, can you perhaps make a function of it? That will make a potential future refactoring easier.

Reply:
Put that function in `core/utils` since it is used across several files. :angel: